src/components/App/App.js

```diff
@@ -7,35 +7,34 @@ import RecyclingTips from '../RecyclingTips/RecyclingTips';
 import SearchBar from '../SearchBar/SearchBar';
 import Footer from '../Footer/Footer';
 import './App.css';
-import Creators from '../Creators/Creators'
-
-// const Creators = React.lazy(() => import("./pages/Creators"));
+import Creators from '../Creators/Creators';
+import NoMatch from '../NoMatch/NoMatch';
 
 const App = () => {
-  const [placeIds, setPlaceIds] = useState([]);
+  const [locationDetails, setlocationDetails] = useState([]);
 
-  const updatePlaceIds = (newPlaceIds) => {
-    setPlaceIds(newPlaceIds);
-    console.log('updatePlaceIds');
+  const updateLocationDetails = (newLocationDetails) => {
+    setlocationDetails(newLocationDetails);
   };
 
   return (
     <main className="app">
       <NavBar />
       <Routes>
         <Route 
-          path="/"
+          path="recycle-me-ui"
           element={<>
             <LandingPage />
-            <Map placeIds={ placeIds } />
-            <SearchBar updatePlaceIds={ updatePlaceIds } />
+            <Map locationDetails={ locationDetails } />
+            <SearchBar updateLocationDetails={ updateLocationDetails } />
+            <RecyclingTips />
```

Comment:
This was a good catch. Recycling Tips was missing from App and you were able to notice that while merging.